src/pages/api/users/index.js

```diff
@@ -1,16 +1,39 @@
-import { createUser } from "../../../../server/db/actions/User";
+import { createUser, getUsers } from "../../../../server/db/actions/User";
 
 export default async function handler(req, res) {
-  const { username, hash } = req.query;
+  if (req.method == "POST") {
```

Comment:
You can delete this whole if statement as it was mainly just a placeholder. Ryder's ticket will cover this functionality